src/scripts/slurm_submit.sh

```diff
@@ -85,8 +85,17 @@ bls_add_job_wrapper
 ###############################################################
 
 datenow=`date +%Y%m%d`
-jobID=`${slurm_binpath}/sbatch $bls_tmp_file` # actual submission
-retcode=$?
+retry=0
+MAX_RETRY=3
```

Comment:
Could we add this as a config variable `slurm_max_submit_retries` [here](https://github.com/osg-bosco/BLAH/blob/v1_18_bosco/config/blah.config.template#L319), defaulting to 0, and reference it via `${slurm_max_submit_retries}`?